torch/onnx/symbolic_opset14.py

```diff
@@ -40,6 +40,13 @@ def reshape(g, self, shape):
 
 @parse_args("v", "v", "v", "v", "v", "i", "f", "f", "i")
 def batch_norm(g, input, weight, bias, running_mean, running_var, training, momentum, eps, cudnn_enabled):
+
+    if torch.is_autocast_enabled() and \
+            not check_args_have_same_dtype([input, weight, bias, running_mean, running_var]) and \
+            sym_help._export_onnx_opset_version < 15:
+        return sym_help._onnx_opset_unsupported_detailed("BatchNormalization", 14, 15,
+                                                         "Tensors with mixed precision are not supported")
```

Comment:
Let's try to make this error message more actionable. Maybe something like:

"All input tensors must have the same dtype. Try turning off autocast or exporting to opset_version=15"